### PR TITLE
browser log

### PR DESCRIPTION
**Pull request by sanchaymittal**

### CODE REVIEW

File: modules/sdk/src/service/index.ts [hunk 1]
```diff
@@ -41,6 +42,10 @@ import {
 
 export { BrowserNode, ERC20Abi, FullChannelState, getBalanceForAssetId, TransferQuote, VectorError };
 
+const LOGDNA_INGESTION_KEY = "86e2bf58c669c293e6525e49eab86a0c";
```
Comment (rhlsthrm) [inline modules/sdk/src/service/index.ts]: We should not commit this to git, I think we need to cycle it now.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: modules/sdk/src/service/index.ts [hunk 2]
```diff
@@ -41,6 +41,14 @@ import {
...
```
Comment (rhlsthrm) [inline modules/sdk/src/service/index.ts]: Why don't we just import this from the config?

File: modules/sdk/src/config.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export default {
+  LOGDNA_INGESTION_KEY: process.env.LOGDNA_INGESTION_KEY ?? "",
```
Comment (rhlsthrm) [inline modules/sdk/src/config.ts]: Hardcode the default here and allow it to be overriden by env var.

File: modules/sdk/src/service/index.ts [hunk 4]
```diff
@@ -514,10 +522,10 @@ export class ConnextSdk {
       }
 
       if (isRecipientAssetInput) {
-        senderAmountUi = formatUnits(senderAmountBn, this.senderChain?.assetDecimals!);
+        senderAmountUi = formatUnits(senderAmountBn.toString(), this.senderChain?.assetDecimals!);
```
Comment (rhlsthrm) [inline modules/sdk/src/service/index.ts:523]: `formatUnits` should work with BN, you don't need to `toString()` it.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]
Small fixes requested.

File: modules/sdk/src/service/index.ts (hunk 4 above)
Comment (sanchaymittal) [reply modules/sdk/src/service/index.ts:523]: i think it's new stuff.
BigNumberish doesn't allow BigNumber

Comment (rhlsthrm) [reply modules/sdk/src/service/index.ts:523]: That's stupid, it's called `BigNumberish` lol. Ok then, proceed with this change.

File: modules/sdk/src/service/index.ts (hunk 2 above)
Comment (sanchaymittal) [reply modules/sdk/src/service/index.ts]: done

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**